main.go

```diff
@@ -20,11 +23,23 @@ var yarnLogDir = flag.String("yarn-logs-dir", "/tmp/logs", "The HDFS path where
 var yarnHistoryDir = flag.String("yarn-history-dir", "/tmp/staging/history/done", "The HDFS path where YARN stores finished job history files. This is the controlled by the hadoop property mapreduce.jobhistory.done-dir.")
 var httpTimeout = flag.Duration("http-timeout", time.Second*2, "The timeout used for connecting to YARN API. Pass values like: 2s")
 var pollInterval = flag.Duration("poll-interval", time.Second*2, "How often should we poll the job APIs. Pass values like: 2s")
+var enableDebug = flag.Bool("pprof", false, "Enable pprof debugging tools at /debug.")
 
 var jt *jobTracker
 
 var rootPath, staticPath string
 
+var mux *web.Mux
```

Comment:
When you `import "net/http/pprof"`, it binds to the default HTTP handler as a side-effect. So you can't import the package and have them optionally enabled unless you use a distinct server object.

Documentation here: https://golang.org/pkg/net/http/pprof/